docs/linkButton.md

```diff
@@ -0,0 +1,60 @@
+# LinkButton
+
+### Usage
+
+```jsx
+import { LinkButton } from 'radiance-ui';
+
+<React.Fragment>
+  <LinkButton.Container>
+    <LinkButton href="#">
+        Primary
+    </LinkButton>
+
+    <LinkButton buttonType="secondary" href="#">
+      Secondary
+    </LinkButton>
+
+    <LinkButton buttonType="tertiary" href="#">
+      Tertiary
+    </LinkButton>
+
+    <LinkButton disabled href="#">
+      I am Disabled
+    </LinkButton>
+  </LinkButton.Container>
+</React.Fragment>;
+```
+
+#### React Router Link
+
+```jsx
+import { LinkButton } from 'radiance-ui';
+import { Link } from 'react-router';
+
+<LinkButton to="/somepath" renderedTag={Link}>
+  Router Link
+</LinkButton>;
+```
+
+<!-- STORY -->
+
+### Proptypes
+
+| prop        | propType        | required | default  | description                                                                                                                  |
+| ----------- | --------------- | -------- | -------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| renderedTag | string, element | no       | "a"      | Specifies the tag be rendered.                                                                                               |
```

Comment:
Random renaming opportunity here. I've seen a lot of react tools use the `as` prop for replacing what tag is rendered (e.g. [emotion](https://emotion.sh/docs/styled#as-prop), semantic-ui-react)

Reply:
@snags88 - yaaaaas! way better than `renderedTag`. Swapped out in 7fc4000.

Your prop naming is woke af... meaning `as={f***}`  😂 

I'll see myself out.